test.py

```diff
@@ -22,5 +22,5 @@ def bubbleSort(array):
 
 bubbleSort(data)
 
-# print('Sorted Array in Ascending Order:')
+print('Sorted Array in Ascending Order:')
 print(data)
```

Comment:
[test.py, line 26]: The code prints the sorted array to the console, which may not be the desired behavior in all cases. It would be better to return the sorted array from the function and let the caller decide what to do with it.